guides/release/in-depth-topics/rendering-values.md

```diff
@@ -71,6 +71,25 @@ or via a property on some object
 ```handlebars
 <this.someComponent />
 ```
+```ts
```

Comment:
There shouldn't be any TypeScript in the guides (yet). There is a whole process that is underway to add a typescript version of the guides but until that is done then you should write all your backing JS as JS 👍